vector/src/main/java/im/vector/app/features/roomprofile/RoomProfileViewModel.kt

```diff
@@ -67,23 +70,27 @@ class RoomProfileViewModel @AssistedInject constructor(
 
     companion object : MavericksViewModelFactory<RoomProfileViewModel, RoomProfileViewState> by hiltMavericksViewModelFactory()
 
-    private val room = session.getRoom(initialState.roomId)!!
+    private val session = sessionHolder.getSafeActiveSession()
+    private val room = session?.getRoom(initialState.roomId)
+    private val myUserId = session?.myUserId ?: ""
 
     init {
-        val flowRoom = room.flow()
-        observeRoomSummary(flowRoom)
-        observeRoomCreateContent(flowRoom)
-        observeBannedRoomMembers(flowRoom)
-        observePermissions()
-        observePowerLevels()
+        room?.also { room ->
```

Comment:
this view model is now in an invalid state where the room is missing, we would need to either provide a real `Session` to the tests or setup mocked sessions (and possibly other dependencies) for `androidTest` tests